### PR TITLE
ACP2E-269: UPS shipping rate calculation error

### PR DESCRIPTION
**Pull request by thiaramus**
## Purpose of this pull request

_Describe the goal and the type of changes this pull request covers. Tell us what changes you are making and why._

This pull request (PR) is to clarify that Magento does not support shipping cost calculation by dimensional weight for FedEx and UPS out of the box.

## Magento release version

_Which Magento release(s) are affected by the content changes: 2.4, 2.3? Specify a patch release number, if applicable._

## Product editions

_Is this update specific to a product edition: Magento Open Source only, Adobe Commerce only?_

- Both

_Is this update specific to an installed feature extension: B2B extension, other feature set (please, specify)?_

- N/A

## Affected documentation pages

_List HTML links for affected pages on <https://docs.magento.com/user-guide/>._

- https://docs.magento.com/user-guide/shipping/ups.html
- https://docs.magento.com/user-guide/shipping/fedex.html

## Links to Magento source code or PRs

_OPTIONAL. If this pull request references a file in a Magento codebase repository or a code PR, add it here._

- https://jira.corp.magento.com/browse/ACP2E-269

## Additional information

_OPTIONAL. What other information can you provide?_

### CODE REVIEW

File: src/shipping/fedex.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ FedEx is one of the world’s largest shipping service companies, providing air,
 *FedEx Shipping Options at Checkout*
 
 {:.bs-callout-info}
-FedEx now uses [dimensional weight]({% link shipping/weight-dimensional.md %}) to determine some shipping rates.
+FedEx can use [dimensional weight]({% link shipping/weight-dimensional.md %}) to determine some shipping rates. However, Magento only supports weight based shipping cost calculation.
```
Comment (leslietilling) [inline src/shipping/fedex.md:11]:
```suggestion
FedEx can use [dimensional weight]({% link shipping/weight-dimensional.md %}) to determine some shipping rates. However, Adobe Commerce and Magento Open Source support only weight-based shipping cost calculation.
```

File: src/shipping/ups.md [hunk 2]
```diff
@@ -4,7 +4,8 @@ title: UPS
 
 United Parcel Service offers domestic and international shipping services by land and air to more than 220 countries.
 
-UPS uses [dimensional weight]({% link shipping/weight-dimensional.md %}) to determine some shipping rates.
+{:.bs-callout-info}
+UPS can use [dimensional weight]({% link shipping/weight-dimensional.md %}) to determine some shipping rates. However, Magento only supports weight based shipping cost calculation.
```
Comment (leslietilling) [inline src/shipping/ups.md:8]:
```suggestion
UPS can use [dimensional weight]({% link shipping/weight-dimensional.md %}) to determine some shipping rates. However, Adobe Commerce and Magento Open Source support only weight-based shipping cost calculation.
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Edits for branding, etc.

**Comment by thiaramus** [comment]
Thank you for the corrections, @leslietilling !

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by dobooth** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by m2-github-services** [comment]
@dobooth the Pull Request is successfully imported.

**Pull request merged**